.github/workflows/r.yml

```diff
@@ -36,13 +36,17 @@ jobs:
   test:
     runs-on: ${{ matrix.config.os }}
     name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    defaults:
+      run:
+        shell: bash
```

Comment:
This is necessary because otherwise `R` is an alias to `Invoke-History` on PowerShell...